graphql_compiler/query_pagination/query_splitter.py

```diff
@@ -42,21 +33,23 @@ def split_into_page_query_and_remainder_query(schema_info, query_ast, parameters
         raise AssertionError(
             u"Could not split query {} into pagination queries for the next page"
             u" of results, as the number of pages {} must be greater than 1: {}".format(
-                query_ast, num_pages, parameters
+                query.query_ast, num_pages, query.parameters
             )
         )
 
-    parameterized_queries = generate_parameterized_queries(schema_info, query_ast, parameters)
+    # TODO propagate advisories to top-level
```

Comment:
This will require an API change in the query splitter, which is a bit out of scope for this PR.

Reply:
I assume next PR?